kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    1.0.3
+    1.0.4
```

Comment:
Incrementing the patch version right here is **the worst thing you can do**. For added functionality that's still backwards compatible you should increment the minor version.